rpm-package/restart.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+NAME=jellyfin
+restart_cmd="/usr/bin/systemctl restart ${NAME}"
+echo "sleep 2; sudo $restart_cmd > /dev/null 2>&1" | at now > /dev/null 2>&1
```

Comment:
Why is the `sleep 2` needed?

Reply:
I reused most of the debian package also [this script](https://github.com/jellyfin/jellyfin/blob/f8a720d3d8adbdb1f092a42e592dae37ba3f25bb/debian/bin/restart.sh#L15) i assume it's to show the "server is restarting" message in the web-ui without getting killed to quickly.

Reply:
Yea that bit was straight outta upstream, we might eliminate it entirely soon.